unlock-app/src/pages/dashboard.js

```diff
@@ -30,13 +30,14 @@ export const Dashboard = ({account, network, locks}) => {
 Dashboard.displayName = 'Dashboard'
 
 Dashboard.propTypes = {
-  account: UnlockPropTypes.account.isRequired,
+  account: UnlockPropTypes.account, // We can't mark this as required because it's undefined when the dashboard is loading
```

Comment:
Yeah, I think we need to fix that in another PR. I think we actually need to keep it required but not render if it is missing ;)